### PR TITLE
Prefix version suspension

### PR DESCRIPTION
**Pull request by krisis**

### CODE REVIEW

**Review by harshavardhana — APPROVED** [review]

File: cmd/version-enable.go [hunk 1]
```diff
@@ -28,13 +29,23 @@ import (
 	"github.com/minio/pkg/console"
 )
 
+var versionEnableFlags = []cli.Flag{
+	cli.StringFlag{
+		Name:  "excluded-prefixes",
```
Comment (vadmeste) [inline cmd/version-enable.go:34]: s/excluded-prefixes/exclude-prefixes ?

File: cmd/version-enable.go [hunk 2]
```diff
@@ -47,6 +58,13 @@ FLAGS:
 EXAMPLES:
   1. Enable versioning on bucket "mybucket" for alias "myminio".
      {{.Prompt}} {{.HelpName}} myminio/mybucket
+
+  2. Enable versioning on bucket "mybucket" while excluding versioning on a few select prefixes.
+     {{.Prompt}} {{.HelpName}} myminio/mybucket --excluded-prefixes "app1/*/_temporary/,app2/*/_staging/"
```
Comment (vadmeste) [inline cmd/version-enable.go:63]: String array is easier for the users I guess:
`--exclude-prefix "app1/*/_temporary/" --exclude-prefix "app2/*/_staging/"`

**Review by vadmeste — COMMENTED** [review]
Nit comments

File: cmd/version-enable.go (hunk 1 above)
Comment (harshavardhana) [reply cmd/version-enable.go:34]: Its written to mimic the XML @vadmeste

File: cmd/version-enable.go (hunk 2 above)
Comment (harshavardhana) [reply cmd/version-enable.go:64]: a comma-separated is fine for now I think

**Review by kannappanr — APPROVED** [review]
LGTM

**Pull request merged**